### ISSUE TITLE
Add Fortmatic, Portis, Legder and Trezor DApp Connectors

### ISSUE THREAD

**Issue by samparsky** [ConsenSys/rimble-ui#350]
**Is your feature request related to a problem? Please describe.**
Additional DApp connectors apart from Metamask for Fortmatic, Portis, Legder and Trezor

*Describe the solution you'd like**
Should have similar implementation to the current Metamask button.

**Describe alternatives you've considered**
None

**Additional context**
<img width="782" alt="Screenshot 2019-07-25 at 9 35 26 AM" src="https://user-images.githubusercontent.com/8148384/61859185-8f705100-aebf-11e9-97d1-2cc36d8f6671.png">

**Comment by gesquinca** [comment]
@samparsky Can you provide more information on this? 

Are you wanting this library to integrate more pre-styled buttons for Fortmatic, Portis, Legder and Trezor, or something else?

**Comment by samparsky** [comment]
Yes more pre-styled buttons for Fortmatic, Portis, Legder and Trezor

**Comment by gesquinca** [comment]
@samparsky Awesome, we'll add it to the backlog. Thanks for submitting this!
